torch/csrc/jit/script/schema_matching.cpp

```diff
@@ -199,6 +199,16 @@ Value* tryCreateList(
       return nullptr;
     list_ctor.push_back(av);
   }
+  for (const auto& item : list_ctor) {
```

Comment:
I don't think we want to error here in schema matching because we're short-circuiting the normal error reporting code, and just because this fails this specific List matching doesn't mean there's another operator that won't be matched